programs/fileio.c

```diff
@@ -611,7 +611,7 @@ FIO_openDstFile(FIO_prefs_t* const prefs,
                && strcmp (srcFileName, stdinmark)
                && strcmp(dstFileName, nulmark) ) {
```

Comment:
If `UTIL_chmod()` guarantees that it will only applies permissions to regular files, then there is no more need to check if `dstFileName` == `nulmark`, simplifying this test.

note : conditions on `srcFileName` must still be checked !